lib/bundler/lockfile_parser.rb

```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 
+#--
```

Comment:
This is necessary to make this (internal) comment invisible for RDoc.

Reply:
This comment is about really really old `bundler` versions, I don't think it's worth making it visible to RDoc.

Reply:
> I don't think it's worth making it visible to RDoc.

Exactly my point! The `#--` at the beginning of the comment tells RDoc to ignore it, otherwise, it would be rendered in the main class' docs :)

Reply:
Oooops! I read "visible", not "unvisible"!